roles/katello_devel/meta/main.yml

```diff
@@ -1,4 +1,9 @@
 ---
+vars:
```

Comment:
this will break `katello-devel` pipe, as `vars` don't belong into `meta`.
can you set these in `defaults/main.yml` instead please?

Reply:
Done in 66ca393f53ff85915206cce67206f80425cd35f9